### ISSUE TITLE
[dataio] dataio-pyshovel multiple files (Trac #1921)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1913]
We should be able to shovel over multiple files (specifically a GCD+data).

Migrated from https://code.icecube.wisc.edu/ticket/1921
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:44", 
    "description": "We should be able to shovel over multiple files (specifically a GCD+data).", 
    "reporter": "david.schultz", 
    "cc": "olivas", 
    "resolution": "fixed", 
    "_ts": "1550067284370534", 
    "component": "combo core", 
    "summary": "[dataio] dataio-pyshovel multiple files", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-12-07T19:56:22", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170511T20:44:57: david.schultz changed attachment from "" to "diff"

**Comment by icecube-trac** [comment]
Trac update at 20170511T20:48:53: 
* david.schultz commented:

> Attached diff that sort-of makes this work, but the frame mixing between files is broken.
> 
> I'm not sure of a great way to solve mixing without #1423, which was closed as wontfix.  Promoting shovelio to a separate project and moving dataio-shovel to it might work.


* david.schultz changed cc from "" to "olivas"

**Comment by icecube-trac** [comment]
Trac update at 20170511T21:39:51: olivas commented:

> I have no problem re-opening #1423.

**Comment by icecube-trac** [comment]
Trac update at 20170516T21:55:49: david.schultz changed milestone from "Long-Term Future" to "July 2017"

**Comment by icecube-trac** [comment]
Trac update at 20170629T21:17:24: 
* david.schultz changed status from "new" to "closed"
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 2927:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2927"
> merge c++ and pybinding I3File. eliminate python fileadaptor. merge shovelio I3FrameSequence. integrate I3FrameSeqeuence into dataio-pyshovel, allowing shoveling over multiple files. closes #1423. closes #1913
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:55:24: nwhitehorn commented:

> In changeset 169219:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169219"
> merge c++ and pybinding I3File. eliminate python fileadaptor. merge shovelio I3FrameSequence. integrate I3FrameSeqeuence into dataio-pyshovel, allowing shoveling over multiple files. closes #1423. closes #1913
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:44: 
* olivas commented:

> Milestone July 2017 deleted


* olivas changed milestone from "July 2017" to ""
